pkg/cover/instrument.go

```
@@ -409,3 +392,18 @@ func checkCacheDir(p string) error {
 	}
 	return nil
 }
+
+func injectGlobalCoverVarFile(ci *CoverInfo, content string) error {
+	coverFile, err := os.Create(filepath.Join(ci.Target, ci.GlobalCoverVarImportPath, "cover.go"))
+	if err != nil {
+		return err
+	}
+	defer coverFile.Close()
+
+	packageName := "package " + filepath.Base(ci.GlobalCoverVarImportPath) + "\n\n"
+
+	_, _ = coverFile.WriteString(packageName)
```

it's not good practice to ignore error